src/main/java/com/northconcepts/datapipeline/examples/cookbook/ParseAnExpression.java

```diff
@@ -0,0 +1,18 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.internal.expression.DefaultExpressionContext;
+import com.northconcepts.datapipeline.internal.expression.Expression;
+import com.northconcepts.datapipeline.internal.parser.Parser;
+
+public class ParseAnExpression {
+
+    public static void main(String[] args) {
+        Expression expression = Parser.parseExpression("2 + 2");
+                
+        DefaultExpressionContext expressionContext = new DefaultExpressionContext();
+        Long evaluatedValue = (Long) expression.evaluate(expressionContext);
+        System.out.println("2 + 2 = " + evaluatedValue);
+        
+        // Learn more about expression language at: https://northconcepts.com/docs/expression-language
```

Comment:
Let's drop  the comment since we'll add content around it in the CMS.

Reply:
Removed